test/s3select_test.cpp

```diff
@@ -704,6 +704,23 @@ TEST(TestS3selectFunctions, between)
   ASSERT_EQ(s3select_result_1,s3select_result_2);
 }
 
+
+TEST(TestS3selectFunctions, not_between)
+{
+  std::string input;
+  size_t size = 128;
+  generate_rand_columns_csv(input, size);
+  const std::string input_query_1 = "select count(0) from stdin where int(_1) not between int(_2) and int(_3);";
+
+  std::string s3select_result_1 = run_s3select(input_query_1,input);
+
+  const std::string input_query_2 = "select count(0) from stdin where int(_1) < int(_2) or int(_1) > int(_3);";
+
+  std::string s3select_result_2 = run_s3select(input_query_1,input);
```

Comment:
the queries seem to be correct (same semantics)
but
using the same query `input_query_1` for both runs.

Reply:
My bad! Updated the commit with the specific builder. Please take a look